### DISCUSSION TITLE
Wiki link broken or incorrect

### DISCUSSION THREAD

**Discussion by ovizii** [thlucas1/spotifyplus_card#9] [General]
https://github.com/thlucas1/spotifyplus_card?tab=readme-ov-file 

![image](https://github.com/user-attachments/assets/e8e45a9f-8a2a-43cf-8020-503690e8f68b)

The "UI-dashboard" part from the URL disappears.
![image](https://github.com/user-attachments/assets/2d0da745-c101-425f-82e9-374db7edcb44)

**Comment by thlucas1** [comment]
Add a placeholder document for [UI Dashboards](https://github.com/thlucas1/spotifyplus_card/wiki/UI-Dashboards) link.

Still working on getting the docs completed.

**Comment by thlucas1** [comment]
The docs are now complete, so all links should function as expected.  Please re-open this discussion if you find any broken links.  
Thanks!
